packages/jaeger-ui/src/model/ddg/transformTracesToPaths.test.js

```diff
@@ -52,7 +60,7 @@ describe('transform traces to ddg paths', () => {
         (result, span) => ({
           ...result,
           [span.processID]: {
-            serviceName: `${span.spanID.split(' ')[0]} service`,
+            serviceName: span.processID,
```

Comment:
This doesn't fully test that the `serviceName`s are populated. change this to
```
serviceName: ${span.processID}-name,
```

Reply:
Done. 

Thanks for the review!